forms-flow-web/src/components/Form/List.js

```diff
@@ -561,6 +580,30 @@ const List = React.memo((props) => {
   );
 });
 
+const ReleaseNoteModal = ({ showModal, onClose, title, content }) => {
+  return (
+      <Modal 
+        show={showModal}
+        onHide={onClose}
+        size="lg"
+        >
+        <Modal.Header style={{color: "#fff", backgroundColor: "#036"}}>
+          <Modal.Title>
+            <h3>{title}</h3>
+          </Modal.Title>
+        </Modal.Header>
+        <Modal.Body>
+            <div dangerouslySetInnerHTML={{ __html: content }} />
```

Comment:
Thoughts on rendering this using the `<Form` component from `react-formio`? As I think that internally does some sanitization etc.
That way we would also be one step closer to moving this into the platform for the PSA team to manage as in the future let them use the form builder to create the notices.

Reply:
@simensma-fresh I have updated API and frontend to support `<Form ` from the `react-formio` . Now in place of HTML string, we will store FORM JSON (generated from the form builder, please check MS team for new database row) in the database, and render it on the release note Modal using `<Form`.